### ISSUE TITLE
Could not register CGFont 'Roboto-Medium'

### ISSUE THREAD

**Issue by rockwotj** [tskulbru/NBMaterialDialogIOS#4]
There is a `NSInternalInconsistencyException` that is thrown when trying to open a dialog using XCode 7.3.1 and iOS 9.1

Please let me know if you need more information

**Comment by tskulbru** [comment]
Do you have a stack trace?

**Comment by rockwotj** [comment]
```
2016-05-29 14:37:29.358 PasswordKeeper[31354:546038] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'The operation couldn’t be completed. (com.apple.CoreText.CTFontManagerErrorDomain error 105 - Could not register the CGFont '<CGFont (0x7fe530e9b250): Roboto-Medium>')'
*** First throw call stack:
(
    0   CoreFoundation                      0x000000010f7a9d85 __exceptionPreprocess + 165
    1   libobjc.A.dylib                     0x00000001117c6deb objc_exception_throw + 48
    2   CoreFoundation                      0x000000010f7a99c9 -[NSException raise] + 9
    3   NBMaterialDialogIOS                 0x000000010f371d52 _TZFC19NBMaterialDialogIOSP33_E761B8D1A7E25D145ABD74353971091D10FontLoader8loadFontfSST_ + 1778
    4   NBMaterialDialogIOS                 0x000000010f372a57 _TFZFE19NBMaterialDialogIOSCSo6UIFont18robotoMediumOfSizeFV12CoreGraphics7CGFloatS0_U_FT_T_ + 71
    5   NBMaterialDialogIOS                 0x000000010f36b987 _TTRXFo__dT__XFdCb__dT__ + 39
    6   libdispatch.dylib                   0x00000001122563eb _dispatch_client_callout + 8
    7   libdispatch.dylib                   0x0000000112241d68 dispatch_once_f + 543
    8   NBMaterialDialogIOS                 0x000000010f371ff7 _TZFE19NBMaterialDialogIOSCSo6UIFont18robotoMediumOfSizefV12CoreGraphics7CGFloatS0_ + 471
    9   NBMaterialDialogIOS                 0x000000010f3721e3 _TToZFE19NBMaterialDialogIOSCSo6UIFont18robotoMediumOfSizefV12CoreGraphics7CGFloatS0_ + 35
    10  NBMaterialDialogIOS                 0x000000010f362844 _TFC19NBMaterialDialogIOS16NBMaterialDialogP33_1614A6FB548B9344A8D350180059BB2E24setupTitleLabelWithTitlefSST_ + 292
    11  NBMaterialDialogIOS                 0x000000010f35d927 _TFC19NBMaterialDialogIOS16NBMaterialDialog10showDialogfTCSo6UIView5titleGSqSS_7contentS1_12dialogHeightGSqV12CoreGraphics7CGFloat_13okButtonTitleGSqSS_6actionGSqFT13isOtherButtonSb_T__17cancelButtonTitleGSqSS_14stackedButtonsSb_S0_ + 2279
    12  NBMaterialDialogIOS                 0x000000010f35cfaa _TFC19NBMaterialDialogIOS16NBMaterialDialog10showDialogfTCSo6UIView5titleGSqSS_7contentS1_12dialogHeightGSqV12CoreGraphics7CGFloat_13okButtonTitleGSqSS_6actionGSqFT13isOtherButtonSb_T__17cancelButtonTitleGSqSS__S0_ + 858
    13  PasswordKeeper                      0x000000010d85c98c _TFC14PasswordKeeper22PasswordViewController11addPasswordfPs9AnyObject_T_ + 940
    14  PasswordKeeper                      0x000000010d85cb76 _TToFC14PasswordKeeper22PasswordViewController11addPasswordfPs9AnyObject_T_ + 54
    15  UIKit                               0x0000000110214a8d -[UIApplication sendAction:to:from:forEvent:] + 92
    16  UIKit                               0x0000000110387e67 -[UIControl sendAction:to:forEvent:] + 67
    17  UIKit                               0x0000000110388143 -[UIControl _sendActionsForEvents:withEvent:] + 327
    18  UIKit                               0x0000000110387263 -[UIControl touchesEnded:withEvent:] + 601
    19  Material                            0x000000010effc95e _TFC8Material14MaterialButton12touchesEndedfTGVs3SetCSo7UITouch_9withEventGSqCSo7UIEvent__T_ + 206
    20  Material                            0x000000010effcb76 _TToFC8Material14MaterialButton12touchesEndedfTGVs3SetCSo7UITouch_9withEventGSqCSo7UIEvent__T_ + 102
    21  UIKit                               0x000000011028799f -[UIWindow _sendTouchesForEvent:] + 835
    22  UIKit                               0x00000001102886d4 -[UIWindow sendEvent:] + 865
    23  UIKit                               0x0000000110233dc6 -[UIApplication sendEvent:] + 263
    24  UIKit                               0x000000011020d553 _UIApplicationHandleEventQueue + 6660
    25  CoreFoundation                      0x000000010f6cf301 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
    26  CoreFoundation                      0x000000010f6c522c __CFRunLoopDoSources0 + 556
    27  CoreFoundation                      0x000000010f6c46e3 __CFRunLoopRun + 867
    28  CoreFoundation                      0x000000010f6c40f8 CFRunLoopRunSpecific + 488
    29  GraphicsServices                    0x0000000112aa8ad2 GSEventRunModal + 161
    30  UIKit                               0x0000000110212f09 UIApplicationMain + 171
    31  PasswordKeeper                      0x000000010d858622 main + 114
    32  libdyld.dylib                       0x000000011228a92d start + 1
    33  ???                                 0x0000000000000001 0x0 + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by tskulbru** [comment]
Fixed with c61a5bdc425d54f386f2accfe89c49644f4fba20
